src/ec/packages/ec-component-description-list/description-list.story.js

```diff
@@ -22,20 +26,35 @@ const prepareList = data => {
   }
   data.items.forEach((item, i) => {
     if (Array.isArray(item.term)) {
-      item.term.forEach((t, ind) => {
-        t = text(`items[${i}].term[${ind}]`, t, tabLabels.required);
+      item.term.forEach((term, ind) => {
```

Comment:
`forEach((term, ind)` the previous `t` is not as bad as `term` named inside a scope of `term` iterable here. Could be `termItem`. Nested indices are usually named with `j`, `k`, etc. rather than `ind`, `inde`(for next one maybe ?)